README.md

```diff
@@ -7,71 +7,116 @@ or decentralization. As it's EVM-compatible (with Solidity/Vyper), 99% of Ethere
 needing to refactor or re-audit any code. zkSync Era also uses an LLVM-based compiler that will eventually enable
 developers to write smart contracts in popular languages such as C++ and Rust.
 
-This directory and its sub-directories contain the source code for the zkEVM fork of the [LLVM](https://llvm.org) framework,
+This directory and its sub-directories contain the source code for the ZKsync fork of the [LLVM](https://llvm.org) framework,
 a toolkit for the construction of highly optimized compilers, optimizers, and run-time environments
 used by the Solidity and Vyper compilers developed by Matter Labs.
 
 ## Overview
 
-Welcome to the zkEVM LLVM project!
+Welcome to the ZKsync LLVM project!
 
 The project has multiple components. The core of the project is
 the `llvm` directory. This contains all of the tools, libraries, and header
 files needed to process intermediate representations and convert them into
 object files. Tools include an assembler, disassembler, bitcode analyzer, and
 bitcode optimizer. These tools are not yet officially supported for third-party front-ends.
-It also contains zkEVM modifications of the standard [LLVM regression tests](https://llvm.org/docs/TestingGuide.html#regression-tests).
+It also contains ZKsync modifications of the standard [LLVM regression tests](https://llvm.org/docs/TestingGuide.html#regression-tests).
 
-The zkEVM back-end is called `EraVM`, and the architecture is called `eravm`.
+The ZKsync back-end is called `EraVM`, and the architecture is called `eravm`.
```

Comment:
But this is about the target, so it must be EraVM.
It makes this sentence a tautology and probably redundant.

Reply:
I was also thinking about that and wanted to check with you. I've removed this line.